pkg/handlers/tdf.go

```diff
@@ -170,3 +214,77 @@ func (h Handler) InspectTDF(toInspect []byte) (TDFInspect, []error) {
 		return TDFInspect{}, []error{fmt.Errorf("tdf format unrecognized")}
 	}
 }
+
+func readBytesFromFile(filePath string) ([]byte, error) {
```

Comment:
I think a file of this signature already exists in `cmd`. Could we please unify all uses of the same function into a function in `pkg/utils`?